pygn-mode.el

```diff
@@ -1125,13 +1125,20 @@ Focus the game after motion."
       (let ((server-script-path (expand-file-name "pygn_server.py" pygn-mode-script-directory)))
         (if (and (file-exists-p server-script-path)
                  (zerop (call-process pygn-mode-python-executable  nil nil nil server-script-path "-version")))
-           (insert (format "[x] Good. The pygn-mode-script-directory ('%s') is good and server script is callable.\n\n" pygn-mode-script-directory))
+           (insert (format "[x] Good. The pygn-mode-script-directory ('%s') is found and the server script is callable.\n\n" pygn-mode-script-directory))
          (insert
           (format
            "[ ] Bad. The pygn-mode-script-directory ('%s') is bad or does not contain working server script (pygn_server.py).\n\n" pygn-mode-script-directory))
          (cl-return-from pygn-mode--run-diagnostic nil)))
+      (dolist (melpa-lib '(uci-mode nav-flash ivy))
+        (if (featurep melpa-lib)
+            (insert (format "[x] Good.  The `%s' library is available.\n\n" melpa-lib))
+          ;; else
```

Comment:
is this a reminder for you or something :laughing:

Reply:
I do it habitually, because I find Lisp `if` otherwise blends together.  Should I remove it?  It appears in several other places.